README.md

```diff
@@ -77,6 +77,7 @@ These environment variables will override other options.
 * `GDT_DOMAIN`: Specify the base URL for live system testing. Falls back to a
```

Comment:
Should we consolidate all environment variables to be discussed in both places? The relationship between DOMAIN and SITEURLS is discussed with the config explanation. Essentially, your SITEURLS should be configured to use the domain token.